### PR TITLE
feat: new library Kafka Streams Topology Design

### PR DESCRIPTION
**Pull request by excalibot**
Kafka Streams Topology Design is an open standard dedicated to the clear and effective description and visualisation of Kafka Stream topologies.
KSTD covers all Kafka Streams DSL operators, as well as essential infrastructure and architectural building blocks.

Submitted by @hartmut-co-uk

Items: Header, Topology, Sub-topology, Apache Kafka Logo, Topic Simple, Topic Default, Topic Advanced, Record, Key-Value Record, Key-Value Record Typed, KStream, KTable, GlobalKTable, filter, map, mapValues, flatMap, flatMapValues, split, merge, join, join (alt), repartition, selectKey, groupBy, reduce/aggregate, count, peek, forEach, stateful, stateful (alt), repartitioned, repartitioned (alt), process, processValues, transformValues, State Store, windowed, Punctuator, Partitioner, Store Type RocksDB, Store Type InMemory, Store Type LRUCache, Store Variant Global, Store Variant Versioned, Store Variant Read-only, Store Attribute Logged, Store Attribute Not Logged, Store Attribute Cached, Store Attribute Not Cached, API, API (alt1), API (alt2), External System, External System (alt1), External System (alt2), Numeric Indicator, Marker Success, Marker Alert, Marker Info, Marker Warn, Inline Note, Callout, Descriptions Box, Legend Landscape, Legend Portrait, KSTD Watermark

[Installation link](https://excalidraw.com/?addLibrary=https%3A%2F%2Fraw.githubusercontent.com%2Fexcalidraw%2Fexcalidraw-libraries%2Fhartmut-co-uk-kafka-streams-topology-design-1725737625562%2Flibraries%2Fhartmut-co-uk%2Fkafka-streams-topology-design.excalidrawlib%3Fraw%3Dtrue)

### CODE REVIEW

File: libraries.json [hunk 1]
```diff
@@ -6351,5 +6351,91 @@
       "Oracle Base Database",
       "Database"
     ]
+  },
+  {
+    "name": "Kafka Streams Topology Design",
+    "description": "Kafka Streams Topology Design is an open standard dedicated to the clear and effective description and visualisation of Kafka Stream topologies.\r\nKSTD covers all Kafka Streams DSL operators, as well as essential infrastructure and architectural building blocks.",
+    "authors": [
+      {
+        "name": "Hartmut Armbruster",
```
Comment (hartmut-co-uk) [inline libraries.json:6360]: Would this also work?
```suggestion
        "name": "Hartmut Armbruster • Thriving.dev",
```

**Comment by hartmut-co-uk** [comment]
Dear Excalidraw team, I'd like to launch my project on Tuesday, Sept 10th on Product Hunt.
It would be fantastic If you could support my launch.

**Comment by dwelle** [comment]
Thanks, merging! ❤️

File: libraries.json (hunk 1 above)
Comment (dwelle) [reply libraries.json:6360]: No, as we'll be migrating to a new database, we need names to just names, not contain any other information. 

But your name will be clickable leading to the website:

image

**Pull request merged**